.github/workflows/weekly.yml

```diff
@@ -30,13 +30,12 @@ jobs:
     strategy:
       matrix:
         flink_branches: [{
-          flink: 1.19-SNAPSHOT,
-          jdk: '8, 11, 17, 21',
+          flink: 2.1-SNAPSHOT,
+          jdk: '17',
           branch: main
-        },
-        {
-          flink: 1.20-SNAPSHOT,
-          jdk: '8, 11, 17, 21',
+        }, {
+          flink: 2.1-SNAPSHOT,
```

Comment:
Add 2.0.0, 2.0-SNAPSHOT and 2.1-SNAPSHOT CIs.

Reply:
Done.